src/components/InputTodo.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react"
```

Comment:
(Optional) Consider adding functionality that makes sure the task name is correct (ex: a series of commas with no text should not be accepted)